configure.ac

```diff
@@ -624,7 +624,7 @@ if test x"$magick6" = x"yes"; then
   LIBS="$LIBS $MAGICK_LIBS"
   CFLAGS="$CFLAGS $MAGICK_CFLAGS"
 
-  AC_CHECK_FUNCS([InheritException AcquireExceptionInfo SetImageProperty SetImageExtent AcquireImage GetVirtualPixels ResetImageProfileIterator ResetImageAttributeIterator ResetImagePropertyIterator MagickCoreGenesis SetImageOption BlobToStringInfo OptimizePlusImageLayers OptimizeImageTransparency])
+  AC_CHECK_FUNCS([InheritException AcquireExceptionInfo SetImageProperty SetImageExtent AcquireImage GetVirtualPixels ResetImageProfileIterator ResetImageAttributeIterator ResetImagePropertyIterator MagickCoreGenesis SetImageOption BlobToStringInfo OptimizePlusImageLayers OptimizeImageTransparency QuantizeImages])
```

Comment:
I think we probably don't need to test for `QuantizeImages`. It looks like it's in magick6, magick7 and graphicsmagick. The only difference is the magick6 / magick7 change to exception handling, which you've accounted for.